tests/conftest.py

```diff
@@ -44,3 +44,8 @@ def vouched_user(stored_users):
 @pytest.fixture
 def private_user(stored_users):
     return stored_users['private']
+
+
+@pytest.fixture
+def unvouched_user(stored_users):
```

Comment:
I was meaning to use the existing `new_user` but I didn't realise this would require additional registration steps. Ideally there'd be a way to create a users in various states via an API or similar. Without this, I'm fine with storing users as we do already. Also, I think I prefer your original name of 'unvouched'.